frame.go

```diff
@@ -1962,6 +1994,16 @@ func appendLong(p []byte, n int64) []byte {
 	)
 }
 
+func (f *framer) writeCustomPayload(customPayload *map[string][]byte) {
+	if len(*customPayload) > 0 {
+		if f.proto < protoVersion4 {
+			panic("Custom payload is not supported with version V3 or less")
+		}
+		f.payload()
```

Comment:
Doing this here has no impact as the header flags are already written

Reply:
Good catch, I realized that and moved it to [writeFrame](https://github.com/gocql/gocql/pull/1182/files#diff-38a0fa12b0c511105c3a411f0df3e318R842) but forgot to remove it from here